packages/venia-concept/src/components/ErrorView/errorView.js

```diff
@@ -4,8 +4,8 @@ import { loadingIndicator } from 'src/components/LoadingIndicator';
 
 const messages = new Map()
     .set('loading', loadingIndicator)
-    .set('notFound', '404 Not Found')
-    .set('internalError', '500 Internal Server Error');
+    .set('notFound', 'That page could not be found. Please try again.')
+    .set('internalError', 'Something went wrong. Please try again.');
```

Comment:
Honestly, I don't think this change helps much—if anything, it makes it more obscure. If we want to make the message friendlier, let's do it right with a real design and some imagery. Until then, I'd rather leave it as is.

Reply:
Makes sense. I do think the `500 Internal Server Error` is misleading though, especially since that codepath can be executed without a `500` actually occurring.

Also misleading is that these are server-side error codes, when the problem (aside from the 404) can be in the client.

Reply:
Sure. They're not literally server error codes; they're intended to mimic what you'd see if this were a server-rendered app, so we know what case we're in. Ultimately, I don't mind changing the copy if it bothers you.